python/ray/_private/runtime_env/mpi.py

```diff
@@ -0,0 +1,76 @@
+import sys
+import logging
+import argparse
+from typing import List, Optional
+from ray._private.runtime_env.context import RuntimeEnvContext
+from ray._private.runtime_env.plugin import RuntimeEnvPlugin
+import subprocess
+
+default_logger = logging.getLogger(__name__)
+
+
+class MPIPlugin(RuntimeEnvPlugin):
+    priority = 90
```

Comment:
what is this for?

Reply:
couldn't find the usage

Reply:
I think it's the ordering of applying the plugin? @architkulkarni  is it right?

Reply:
That's right!